apps/app/src/components/QuestionItem/QuestionItem.tsx

```diff
@@ -33,6 +36,7 @@ export const QuestionItem = ({
 			{leftSection}
 			<MarkdownContent source={mdxContent} />
 			<div className="mt-1 flex min-w-max flex-col items-center md:ml-4 md:items-end">
+				{withTechnology && <QuestionTechnology technology={technology} />}
```

Comment:
Skoro mamy `leftSection` to nie lepiej byłoby mieć też `rightSection`? :D

Reply:
ma to sens, przebuduję to